patches/node/fix_comment_out_incompatible_crypto_modules.patch

```diff
@@ -28,23 +28,29 @@ index 91cb94d8dbe9db0adbee5e005649188e1ccbcbf9..2429ebf6035de392c6bac18c0b924b99
    BIGNUM* bn_p =
        BN_bin2bn(reinterpret_cast<const unsigned char*>(p), p_len, nullptr);
    BIGNUM* bn_g = BN_new();
-@@ -5214,6 +5216,7 @@ bool DiffieHellman::Init(const char* p, int p_len, int g) {
- 
+@@ -5215,18 +5217,18 @@ bool DiffieHellman::Init(const char* p, int p_len, int g) {
  bool DiffieHellman::Init(const char* p, int p_len, const char* g, int g_len) {
    dh_.reset(DH_new());
-+#if 0
    if (p_len <= 0) {
-     BNerr(BN_F_BN_GENERATE_PRIME_EX, BN_R_BITS_TOO_SMALL);
+-    BNerr(BN_F_BN_GENERATE_PRIME_EX, BN_R_BITS_TOO_SMALL);
++    // BNerr(BN_F_BN_GENERATE_PRIME_EX, BN_R_BITS_TOO_SMALL);
+     return false;
```

Comment:
If we're re-enabling these p_len <= 0 tests here (sans the `BNerr()` call) should we do the same above for other version of DiffieHellman::Init()?

Reply:
yeah good catch we should